end-to-end/cypress/e2e/portal/portal-ui.cy.js

```diff
@@ -28,9 +28,11 @@ describe('portal-ui', () => {
 
       // Static pages are tested separately.
 
+      cy.findByTestId('user-profile-dropdown').contains('User Profile').click();
+
       // login
-      cy.contains('login');
       // Don't click! We shouldn't depend on Globus in tests.
+      cy.findByTestId('login-link').contains('Log In').click();
```

Comment:
Does this test pass without the `click`?

Reply:
Whoops, that was an unintentional result of copy and paste. I'll remove the click here.